test/e2e/snaps/test-snap-confirm.spec.js

```diff
@@ -0,0 +1,94 @@
+const { strict: assert } = require('assert');
+const { withFixtures } = require('../helpers');
+
+describe('Test Snap Confirm', function () {
+  it('can pop up a snap confirm and get its result', async function () {
+    const ganacheOptions = {
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: 25000000000000000000,
+        },
```

Comment:
In another PR we have upgraded ganache and this may require a change like [here](https://github.com/MetaMask/metamask-extension/pull/13282/files#diff-823327c04da09b29b03f70883023f4278ca78c9f84ef40a63bb2a28a30013ed7R18).